src/SprykerEco/Zed/Computop/Business/Payment/Handler/Saver/Init/PayuCeeSingleResponseSaver.php

```diff
@@ -96,9 +96,9 @@ protected function executeSavePaymentResponseTransaction(
         ComputopPaymentComputopTransfer $computopPaymentComputopTransfer,
         ComputopPayuCeeSingleInitResponseTransfer $computopPayuCeeSingleInitResponseTransfer
     ): void {
-        $this->savePaymentComputopEntity($computopPaymentComputopTransfer, $computopPayuCeeSingleInitResponseTransfer->getHeader());
+        $this->savePaymentComputopEntity($computopPaymentComputopTransfer, $computopPayuCeeSingleInitResponseTransfer->getHeaderOrFail());
```

Comment:
please check all such cases across PR
so I won't comment every one
we need to update affected public API speciications
please make sure you didn't break existing behavior (like requiring something which was not required before)

Reply:
Fixed